продвинутый курс javascript/homework2/javascript/script.js

```diff
@@ -0,0 +1,169 @@
+class GoodsItem {
+    constructor(name, price, cover) {
+        this.cover = cover;
+        this.name = name;
+        this.price = price;
+    }
+    render() {
+        return `<div class='featuredItem'>
+                    <div class='featuredImgWrap'>
+                        <div class='featuredBuy'>
+                            <button>
+                                <img  src='images/addToCart.png' alt="">Add to Cart
+                            </button>
+                        </div>
+                        <img src=${this.cover}>
+                    </div>
+                    <div class='featuredNameAndPrice'>
+                        <div class='featuredItemName'>
+                            ${this.name}
+                        </div>
+                        <div class='featuredItemPrice'>
+                            $${this.price.toFixed(2)}
+                        </div>
+                    </div>
+                </div>`;
+    }
+}
+
+class GoodsList {
+    constructor() {
+        this.goods = [];
+    }
+
+    fetchGoods() {
+        this.goods = [
+            {
+                name: 'MANGO PEOPLE T-SHIRT',
+                price: 60.00,
+                cover: 'images/featuredItem1.jpg'
+    },
+            {
+                name: 'MANGO PEOPLE T-SHIRT',
+                price: 65.00,
+                cover: 'images/featuredItem2.jpg'
+    },
+            {
+                name: 'MANGO PEOPLE T-SHIRT',
+                price: 87.00,
+                cover: 'images/featuredItem3.jpg'
+    },
+            {
+                name: 'MANGO PEOPLE T-SHIRT',
+                price: 37.00,
+                cover: 'images/featuredItem4.jpg'
+    },
+            {
+                name: 'MANGO PEOPLE T-SHIRT',
+                price: 52.00,
+                cover: 'images/featuredItem5.jpg'
+    },
+            {
+                name: 'MANGO PEOPLE T-SHIRT',
+                price: 42.00,
+                cover: 'images/featuredItem6.jpg'
+    },
+            {
+                name: 'MANGO PEOPLE T-SHIRT',
+                price: 72.00,
+                cover: 'images/featuredItem7.jpg'
+    },
+            {
+                name: 'MANGO PEOPLE T-SHIRT',
+                price: 62.00,
+                cover: 'images/featuredItem8.jpg'
+    },
+];
+        this.goods = this.goods.map(item => new GoodsItem(item.name, item.price, item.cover));
+    }
+
+    render() {
+        const itemsHtml = this.goods.map(item => item.render());
+        return itemsHtml.join('');
+    }
+
+    totalPrice() {
+        let total = this.goods.reduce(function(sum,item){
```

Comment:
```suggestion
        const total = this.goods.reduce(function(sum,item){
```
кроме прочего тут что-то случилось с разметкой кода